libs/blocks/merch-card/merch-card.js

```diff
@@ -194,6 +195,13 @@ const init = (el) => {
       }
     }
     parseContent(el, altCta, cardType, merchCard);
+    const offerSelection = cardType === 'plans' ? el.querySelector('ul') : null;
+    if (offerSelection) {
+      import('./offer-selection.js')
```

Comment:
shouldn't we await for the import here?

`const {default: {initOfferSelection} } = await import('./offer-selection.js');` or something in line.

Reply:
you mean to avoid 'offer selection' radios appearing after the merch card was attached to dom?
i can make this change if we are ok with having card 'init' async

Reply:
updated